axelrod/fingerprint.py

```diff
@@ -284,6 +284,8 @@ def plot(self, col_map='seismic', interpolation='none', title=None):
             http://matplotlib.org/examples/images_contours_and_fields/interpolation_methods.html
         title : str, optional
             A title for the plot
+        colorbar : bool, optional
+            Choose whether the colorbar should be included or not
```

Comment:
`filename` appears in the docstring but doesn't seem to be in the signature. Is that still required?